### PR TITLE
ovl_Oceff_Wipe6 (Song of Soaring Ocarina Effect)

### PR DESCRIPTION
**Pull request by CrookedPoe**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]

**Review by AngheloAlf — CHANGES_REQUESTED** [review]
I just noted the AnimatedMaterial isn't being extracted yet

File: src/overlays/actors/ovl_Oceff_Wipe6/z_oceff_wipe6.c [hunk 1]
```diff
@@ -22,12 +21,80 @@ const ActorInit Oceff_Wipe6_InitVars = {
...
+
+void OceffWipe6_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    OceffWipe6* this = THIS;
+    f32 z;
+    u8 alpha;
+    s32 i;
+    s32 counter;
+    Vec3f activeCamEye;
+    s32 pad;
+    Vec3f vec;
```
Comment (hensldm) [inline src/overlays/actors/ovl_Oceff_Wipe6/z_oceff_wipe6.c:64]: Looks like this is the result from `Camera_GetSkyboxOffset` so perhaps `skyboxOffset` as a name?

**Review by hensldm — APPROVED** [review]
One small suggestion, but happy enough to approve.

File: include/functions.h [hunk 2]
```diff
@@ -1318,7 +1318,7 @@ s16 Camera_ClearFlags(Camera* camera, s16 flags);
 // void func_800DFFAC(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE2 param_5, UNK_TYPE2 param_6, UNK_TYPE2 param_7);
 // UNK_TYPE4 func_800E007C(Camera* param_1, Camera* param_2);
 // UNK_TYPE4 func_800E01AC(void);
-// void func_800E01B8(void);
+Vec3f* Camera_GetSkyboxOffset(Vec3f* dst, Camera* camera);
```
Comment (engineer124) [inline include/functions.h:1321]:
```suggestion
Vec3f* Camera_GetQuakeOffset(Vec3f* dst, Camera* camera);
```
We found out from OoT that `Camera_GetSkyboxOffset` is wrong and that the real name should actually be `Camera_GetQuakeOffset`

File: src/overlays/actors/ovl_Oceff_Wipe6/z_oceff_wipe6.c (hunk 1 above)
Comment (engineer124) [reply src/overlays/actors/ovl_Oceff_Wipe6/z_oceff_wipe6.c:64]:
```suggestion
    Vec3f quakeOffset;
```
`quakeOffset` would be the proper name for this

File: include/functions.h (hunk 2 above)
Comment (CrookedPoe) [reply include/functions.h]: Acknowledged and modified accordingly.

File: src/overlays/actors/ovl_Oceff_Wipe6/z_oceff_wipe6.c (hunk 1 above)
Comment (CrookedPoe) [reply src/overlays/actors/ovl_Oceff_Wipe6/z_oceff_wipe6.c]: Acknowledged and modified accordingly.

**Comment by CrookedPoe** [comment]
Corrupted branch; Starting clean.

**Pull request closed without merging**